src/main/java/com/avispl/symphony/dal/logitech/collabos/LogitechCollabOsCommunicator.java

```diff
@@ -325,12 +329,35 @@ private void retrieveRoomSightsData() {
 		}
 	}
 
+	/**
+	 * Retrieves device sights data from the device.
+	 */
+	private void retrieveDeviceSightsData() {
+		try {
+			JsonNode response = doGet(LogitechCommand.INSIGHTS_DEVICE.getName(), JsonNode.class);
+			if (response != null && !response.get(LogitechConstant.CODE).isNull() && 200 == response.get(LogitechConstant.CODE).intValue() && response.has(LogitechConstant.RESULT)) {
```

Comment:
response.get(LogitechConstant.CODE) could be extracted into a variable and reused